hack/update-deps.sh

```diff
@@ -22,7 +22,7 @@ source $(git rev-parse --show-toplevel)/vendor/github.com/tektoncd/plumbing/scri
 
 cd ${REPO_ROOT_DIR}
 
-KNATIVE_VERSION="release-1.9"
+KNATIVE_VERSION="release-1.10"
```

Comment:
curious how you figured out which version tag corresponded to the tag release-1.10? When I looked, I saw a branch release-1.10 on the github repo, and a few versions on pkg.go.dev, but couldn't figure out which was which.

Reply:
I am not looking at the tag but at the branches (as doesn't use tags at all). `go get …` and `go mod …` work with branches as well as tags 👼🏼, and not looking at pkg.go.dev at all either 🙃

Reply:
good to know, thanks!